WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalsStep.swift

```diff
@@ -16,7 +16,6 @@ final class VerticalsStep: WizardStep {
 
     private func didSelect(_ vertical: SiteVertical) {
         creator.vertical = vertical
-        // Will have to change to the basic information instead
-        delegate?.wizard(self, willNavigateTo: WebAddressStep.identifier)
+        delegate?.wizard(self, willNavigateTo: SiteInformationStep.identifier)
```

Comment:
This is what prompted my comment in `SiteInformationStep` - I think we should be telling the wizard whether we succeeded or failed, or _perhaps_ whether we should advance or rewind.